Header header
time timestamp
string fsm_state
float64[2] position
float64[2] filtered
float64 heading
time last_moved
bool in_rescue
bool onRoad
int8 rescue_class
# path = list() 